smarty/forcefield_utils.py

```diff
@@ -388,25 +392,38 @@ def merge_system( topology0, topology1, system0, system1, label0="AMBER system",
     ----------
     topology : OpenMM Topology
     system : OpenMM System
-    positions: ???
+    positions: unit.Quantity position array
     """
 
     #Load OpenMM Systems to ParmEd Structures
     structure0 = parmed.openmm.load_topology( topology0, system0 )
     structure1 = parmed.openmm.load_topology( topology1, system1 )
-    from simtk.openmm.app import Topology
+
     #Merge parameterized Structure
     structure = structure0 + structure1
     topology = structure.topology
 
+    #Concatenate positions arrays
+    positions0 = np.array(positions0._value)
+    positions1 = np.array(positions1._value)
+    coordinates = np.vstack((positions0,positions1))
+    natoms = len(coordinates)
+    positions = np.zeros([natoms,3], np.float32)
+    for index in range(natoms):
+        (x,y,z) = coordinates[index]
+        positions[index,0] = x
+        positions[index,1] = y
+        positions[index,2] = z
+    positions = unit.Quantity(positions, unit.angstroms)
```

Comment:
Is this perhaps dangerous, @limn1 ? It looks like you assume that the initial position arrays are in angstroms. Should line 417 be `unit.Quantity(positions, positions0._units)` or some such? And a check around 407/408 that both are in angstroms, or put them both in angstroms before using?